pkg/package_archive/README.md

```diff
@@ -0,0 +1,5 @@
+# package_archive
```

Comment:
Couldn't it be called `package_reader` or something else? `pub_package_reader` or `package_validator`?